forge16/src/main/java/com/envyful/api/forge/player/ForgeEnvyPlayer.java

```diff
@@ -28,11 +28,6 @@ protected ForgeEnvyPlayer(SaveManager<ServerPlayerEntity> saveManager, ServerPla
         this.parent = player;
     }
 
-    @Override
-    public UUID getUuid() {
-        return this.parent.getUUID();
-    }
-
     @Override
     public UUID getUniqueId() {
         return this.parent.getUUID();
```

Comment:
Didnt you also say you were removing this method too?

Reply:
That's the Forge method for gtetting the player's UUID